src/components/Breadcrumbs.tsx

```diff
@@ -43,56 +48,49 @@ const Breadcrumbs: React.FC<IProps> = ({
       : ""
 
     return {
-      fullPath: slugChunk.slice(0, idx + 2 + startDepth).join("/") + "/",
```

Comment:
I think we need this final `/`

It is not showing as active [here](https://ethereumorgwebsitedev01-tylerapfleddererethereumo29374.gatsbyjs.io/en/contributing/translation-program/acknowledgements/):
![image](https://user-images.githubusercontent.com/468158/207969802-5d1a7466-e71e-43d3-ac1d-3c2262876f51.png)

Reply:
Whoops, Got it!